pkgs/development/libraries/libgpg-error/default.nix

```diff
@@ -16,6 +16,7 @@ stdenv.mkDerivation rec {
 
   # If architecture-dependent MO files aren't available, they're generated
   # during build, so we need gettext for cross-builds.
+  crossAttrs.nativeBuildInputs = [ gettext buildPackages.stdenv.cc ];
```

Comment:
Need to get rid of cross attrs and some other things. I can force push to fix or just skip for now.